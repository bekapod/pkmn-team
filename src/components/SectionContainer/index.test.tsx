import React from "react";
// tslint:disable-next-line:no-implicit-dependencies
import renderer from "react-test-renderer";
import SectionContainer from ".";

describe("<SectionContainer />", () => {
  it("renders without crashing", () => {
    const tree = renderer.create(
      <SectionContainer>A section of content</SectionContainer>
    );

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
