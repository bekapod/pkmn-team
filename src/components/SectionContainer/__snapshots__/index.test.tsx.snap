// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionContainer /> renders without crashing 1`] = `
<div
  className="SectionContainer-xjlkvu-0 jZbaLj"
>
  A section of content
</div>
`;
